blog/_posts/2017-12-11-Thanks-for-diversifying-tech-with-us.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Thanks for diversifying tech with us!
+layout: post
+created_at: Mon December 11 2017
+permalink: blog/2017-12-11-Thanks-for-diversifying-tech-with-us
+current: blog
+author: Inês Coelho
+twitter: ines_opcoelho
+categories:
+- news
+---
+
+For the past five years, every (north hemisphere) summer, RGSoC has been offering the opportunity for women (transgender and cisgender) and non-binary people from all over the world to work on an Open Source project. This wouldn't be possible without the help and love of our wonderful community. 
+
+![Together we are changing the world!](/img/blog/2017/2017-12-12-change-the-world.jpg)
+<div class="image-credits">Our 20 teams - the lives we touched on the 2017 edition of RGSoC (Image: Inês Coelho)</div>
+
+### Our sponsors and donors
+
+Our program is based on a crowdfunding and sponsorship campaign that runs all year round, whose money is used to finance our teams. This year we were able to fund 16 out of our 20 participating teams of two! 
```

Comment:
could you remove “that runs all year around” (since it's not 100% true as we do officially “open” it every year)? the sentence would then change to “[...] based on a crowdfunding and sponsorship campaign whose money [...]”

Reply:
I believe “we were able to fund 10 out of our 20 participating teams” might sound confusing to people, maybe if we replaced `fund` with `financially support` it would be a bit clearer. What do you think?

Reply:
Seems good to me, thanks for the suggestion!